04.ls/list.rb

```diff
@@ -22,11 +22,11 @@ def print_list
 
   def fetch_items
     flags = @options.dotmatch_flag
-    items = Dir.glob('*', flags).sort
-    items = @options.reverse? ? items.reverse : items
+    file_names = Dir.glob('*', flags)
+    sorted_file_names = @options.reverse? ? file_names.sort.reverse : file_names.sort
```

Comment:
```suggestion
    file_names = Dir.glob('*', flags).sort
    sorted_file_names = @options.reverse? ? file_names.reverse : file_names
```

の方がよりDRYになりますね。

おそらく

```ruby
file_names = Dir.glob('*', flags).sort
```

だとソートしてるから`sorted_file_names`にしないといけないのでは？と考えたからだと予想しますが、この場合、

```ruby
    # これは仮のソート（要件を満たすためのソートは未完了）
    file_names = Dir.glob('*', flags).sort
    # これが最終的なソート
    sorted_file_names = @options.reverse? ? file_names.reverse : file_names
```

と考えれば `file_names = Dir.glob('*', flags).sort` でも問題ありません。